tests/functional/adapter/list_relations_tests/test_pagination.py

```diff
@@ -122,8 +121,8 @@ def test__snowflake__list_relations_without_caching_termination(self, project):
         schemas = project.created_schemas
 
         for schema in schemas:
-            schema_relation = {"database": database, "schema": schema}
-            kwargs = {"schema_relation": schema_relation}
+            schema_relation = SnowflakeRelation.create(database=database, schema=schema)
+            kwargs = {"schema_relation": schema_relation.render()}
```

Comment:
Is there a reason to render here? Are we testing if the macro can take a relation object or a string (or both I suppose)?

Reply:
you can't pass a proper python object to run-operation AFAIK so this is an easy workaround